test/playwright/schema.gql

```diff
@@ -0,0 +1,1096 @@
+"""Exposes a URL that specifies the behaviour of this scalar."""
```

Comment:
Should this file be checked in since it's generated?  Seems like we should leave it out.

Reply:
Yeah, I think this can be dropped.  I can add it to the ignore and remove it from the repo here too.  I'll make this change today.